*: add configuration performance lite-init-stats (#13269)
pingcap/docs: statistics.md

- To specify the maximum number of columns that the synchronously loading statistics feature can process concurrently, modify the value of the [`stats-load-concurrency`](/tidb-configuration-file.md#stats-load-concurrency-new-in-v540) option in the TiDB configuration file. The default value is `5`.
- To specify the maximum number of column requests that the synchronously loading statistics feature can cache, modify the value of the [`stats-load-queue-size`](/tidb-configuration-file.md#stats-load-queue-size-new-in-v540) option in the TiDB configuration file. The default value is `1000`.

> **Warning:**
>
> Lightweight statistics initialization is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

> Since v7.1.0. TiDB introduces lightweight statistics initialization. When the value of `lite-init-stats` is true, statistics initialization doesn't load any histogram, TopN, Count-Min Sketch of any index or column into memory. When the value of `lite-init-stats` is false, statistics initialization loads histograms, TopN, Count-Min Sketch of indexes and primary keys into memory but doesn't load any histogram, TopN, Count-Min Sketch of any non-primary key column into memory. When the optimizer needs the histogram, TopN, Count-Min Sketch of some index or column, those needed statistics will be loaded into memory synchronously or asynchronously. Setting `lite-init-stats` to true speeds up statistics initialization and reduces TiDB's memory usage by avoiding unnecessary statistics loading.

Author: ```suggestion
Starting from v7.1.0. TiDB introduces [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) for lightweight statistics initialization. 

- When the value of [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) is `true`, statistics initialization does not load any histogram, TopN, Count-Min Sketch of any index or column into memory. 
- When the value of `lite-init-stats` is `false`, statistics initialization loads histograms, TopN, Count-Min Sketch of indexes and primary keys into memory but does not load any histogram, TopN, Count-Min Sketch of any non-primary key column into memory. When the optimizer needs the histogram, TopN, Count-Min Sketch of some index or column, those needed statistics will be loaded into memory synchronously or asynchronously. 

Setting `lite-init-stats` to `true` speeds up statistics initialization and reduces TiDB memory usage by avoiding unnecessary statistics loading.
```

> **Warning:**
>
> Lightweight statistics initialization is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

Starting from v7.1.0. TiDB introduces [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) for lightweight statistics initialization. 

- When the value of [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) is `true`, statistics initialization does not load any histogram, TopN, Count-Min Sketch of any index or column into memory. 
- When the value of `lite-init-stats` is `false`, statistics initialization loads histograms, TopN, Count-Min Sketch of indexes and primary keys into memory but does not load any histogram, TopN, Count-Min Sketch of any non-primary key column into memory. When the optimizer needs the histogram, TopN, Count-Min Sketch of some index or column, those needed statistics will be loaded into memory synchronously or asynchronously. 

> Setting `lite-init-stats` to `true` speeds up statistics initialization and reduces TiDB memory usage by avoiding unnecessary statistics loading.

Author: ```suggestion
The default value is `false`, which means to disable lightweight statistics initialization. Setting `lite-init-stats` to `true` speeds up statistics initialization and reduces TiDB memory usage by avoiding unnecessary statistics loading.
```
